src/pandas_profiling/profile_report.py

```diff
@@ -124,17 +128,14 @@ def __init__(
                 .update(report_config.dict(exclude_defaults=True))
             )
 
-        if config is not None:
```

Comment:
I suspect this to be a wrong merge as it looks like a recent fix for the report config. @alexbarros FYI

Reply:
I believe it was done intentionally since this check was moved a bit upwards. Still, it can have some unexpected behaviors because the parameters `explorative`, `sensitive`, `dark_mode`, and `orange_mode` will be applied over a predefined config, which may lead to overwriting the configuration.

Reply:
As Barros mentioned, it is intended as previous installations were simply ignoring those inputs. The change of code fix this.